app/services/force/soql/preference_service.rb

```diff
@@ -27,22 +27,28 @@ def buildAppPreferencesFilters(opts)
         # This builds the syntax need for soql to do an
         # OR statement when passing the option of `Vision/Hearing`
```

Comment:
Yeah this comment seems to explain the special Vision/Hearing case you need to handle